repos/system_upgrade/el7toel8/actors/removeoldlog/actor.py

```diff
@@ -0,0 +1,18 @@
+from leapp.actors import Actor
+from leapp.libraries.actor.library import remove_log
+from leapp.tags import IPUWorkflowTag, FactsPhaseTag
+
+
+class RemoveOldLog(Actor):
+    """
+    Removes the old log from the previous Leapp run
+    to ensure that you have only valid and updated logs for debugging.
+    """
+
+    name = 'remove_old_log'
+    consumes = ()
+    produces = ()
+    tags = (IPUWorkflowTag, FactsPhaseTag)
```

Comment:
I am still not sure if we should do it in the FactsPhase or in the InitramPreparationPhase

@pirat89 what do you think?

Reply:
The `InterimPreparationPhaseTag` - yep.